docs/DafnyRef/Grammar.md

```diff
@@ -565,16 +565,16 @@ datatype definitions.
 
 Several Dafny constructs bind one or more variables to a range of possible values.
 For example, the quantifier `forall x: nat | x <= 5 :: x * x <= 25` has the meaning
-"for all integers x between 0 and 5, the square of x is at most 25".
+"for all integers x between 0 and 5 inclusive, the square of x is at most 25".
 Similarly, the set comprehension `set x: nat | x <= 5 :: f(x)` can be read as
-"the set containing the result of applying f to x, for each integer x between 0 and 5".
+"the set containing the result of applying f to x, for each integer x from 0 to 5 inclusive".
 The common syntax that specifies the bound variables and what values they take on
 is known as the *quantifier domain*; in the previous examples this is `x: nat | x <= 5`, 
 which binds the variable `x` to the values `0`, `1`, `2`, `3`, `4`, and `5`.
 
 Here are some more examples.
 
-- `x: byte` (where `type byte = x | 0 <= x < 256`)
+- `x: byte` (where a value of type `byte` has values `x | 0 <= x < 256`)
```

Comment:
I love everything except this change, because I'm not sure that "value `x | …`" is clear at this point in the manual.

Reply:
Revised